src/main/java/org/zalando/nakadi/repository/kafka/KafkaTopicRepository.java

```diff
@@ -328,22 +327,19 @@ public void syncPostBatch(final String topicId, final List<BatchItem> batch, fin
             } catch (final TimeoutException ex) {
                 kafkaFactory.terminateProducer(producer);
                 failUnpublished(batch, "timed out");
-                TracingService.setCustomTags(kafkaPublishingScope.span(), ImmutableMap.<String, Object>builder()
-                        .put("failure_reason", "timed out").build());
-                TracingService.logErrorInSpan(kafkaPublishingScope, ex.getMessage());
+                publishingSpan.setTag("failure_reason", "timed out");
+                TracingService.logErrorInSpan(kafkaPublishingSpan, ex.getMessage());
```

Comment:
please, remove both of this lines and fuether in the code, it does not make sense for the end user